include/net/aws_iot.h

```diff
@@ -201,7 +205,7 @@ struct aws_iot_config {
  *  @return 0 If successful.
  *            Otherwise, a (negative) error code is returned.
  */
-int aws_iot_init(const struct aws_iot_config *const config,
```

Comment:
Why was const removed?

Reply:
To get around the warning when providing `const struct aws_iot_config *const config` to the poll thread as a `void*`. I did change the overall approach now so this is no longer an issue.